mlflow/ml-package-versions.yml

```diff
@@ -114,7 +113,7 @@ keras:
       # keras 2.3.1 is incompatible with tensorflow > 2.2.1 and causes the issue reported here:
       # https://github.com/tensorflow/tensorflow/issues/38589
       "== 2.3.1": ["tensorflow==2.2.1"]
-      "> 2.3.1, != dev": ["tensorflow"]
+      "> 2.3.1": ["tensorflow<2.5.0"]
```

Comment:
https://github.com/tensorflow/tensorflow/blob/5177e09c9f302372137f28ab7798a8a413c352f7/tensorflow/tools/pip_package/setup.py#L105-L107

```python
    # TODO(scottzhu): OSS keras hasn't been formally released yet. 
    # Use keras-nightly at the moment.
    'keras-nightly ~= 2.5.0.dev',
```

Looks like it'll be removed once keras 2.5 is released